src/devices/DCMotor/DCMotor.cs

```diff
@@ -116,20 +116,34 @@ public static DCMotor Create(int speedControlPin, GpioController controller = nu
         }
 
         /// <summary>
-        /// Creates <see cref="DCMotor"/> instance which allows to control speed in both directions.
+        /// Starts the motor by starting pwm with last speed set.
+        /// </summary>
+        public virtual void Start()
+        {
+        }
+
+        /// <summary>
+        /// Stops the motor by stoping pwm. Useful in case of additional safeguards, e.g. limit switch.
+        /// </summary>
+        public virtual void Stop()
+        {
+        }
+
+        /// <summary>
+        /// Creates <see cref="DCMotor"/> instance with 3pin driver using two pins which allows to control speed in both directions.
         /// </summary>
         /// <param name="speedControlChannel"><see cref="PwmChannel"/> used to control the speed of the motor</param>
         /// <param name="directionPin">Pin used to control the direction of the motor</param>
         /// <param name="controller"><see cref="GpioController"/> related to the <paramref name="directionPin"/></param>
         /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
         /// <returns><see cref="DCMotor"/> instance</returns>
         /// <remarks>
-        /// <paramref name="speedControlChannel"/> can be connected to either enable pin of the H-bridge.
-        /// or directly to the input related with the motor (if H-bridge allows inputs to change frequently).
+        /// <paramref name="speedControlChannel"/> should be connected to the one of two inputs
+        /// related with the motor direction (if H-bridge allows inputs to change frequently).
         /// <paramref name="directionPin"/> should be connected to H-bridge input corresponding to one of the motor inputs.
         /// Connecting motor directly to GPIO pin is not recommended and may damage your board.
         /// </remarks>
-        public static DCMotor Create(PwmChannel speedControlChannel, int directionPin, GpioController controller = null, bool shouldDispose = true)
+        public static DCMotor Create3pinDriver(PwmChannel speedControlChannel, int directionPin, GpioController controller = null, bool shouldDispose = true)
```

Comment:
same

Reply:
here specifically I was expecting you'd change description of directionPin and make name less confusing.. also another comment is that I'd personally consider this 2 pin (i.e. 1-wire protocol is actually using 2 wires as they don't consider  ground as a pin)

Reply:
As I mentioned, I took the 3 pin from the L298N driver because it has 3 pins to support one motor, how many of them we will use is another matter, I thought that this way it would be easier to distinguish which driver we are referring to (my has 2 active pins, I also don't consider ground as a pin).
In this case, directionPin does actually change direction, so I left the old name.
We can change the name to input2 but then it would consistently change speedControlChannel to input1, and the type would be a hint what signal will be there?